lib/gitlab/cli_helpers.rb

```diff
@@ -99,16 +101,29 @@ def output_table(cmd, args, data)
       end
     end
 
+    def output_json(cmd, args, data)
+      if data.empty?
+        puts '{}'
+      else
+        case data
```

Comment:
`hash_result = case data` is preferable here instead of repeating the assignment.